.buildkite/pipeline.linux.yml

```diff
@@ -125,7 +125,20 @@ steps:
       os: linux
     plugins:
       - docker#v3.0.1:
-          image: "fandree/rust-vmm-dev"
+          image: "rustvmm/dev:v2"
+          always-pull: true
+
+  - label: "clippy-arm"
+    commands:
+     - cargo clippy --all
```

Comment:
Clippy command should also have deny warnings. That was a miss on my part, the command for both x86 and aarch64 should be:
```cargo clippy --all -- -D warnings```